h2o-core/src/main/java/hex/Model.java

```diff
@@ -2862,25 +2864,26 @@ private InteractionPair(int v1, int v2, String[] v1Enums, String[] v2Enums) {
           final AutoBuffer ab = new AutoBuffer(inputStream);
           @SuppressWarnings("unchecked")
           M model = (M) Keyed.readAll(ab);
+          Keyed.readAll(ab); // CV holdouts frame 
           ab.close();
           return model;
         } 
     }
-
+    
   /**
    * Exports a binary model to a given location.
    * @param location target path, it can be on local filesystem, HDFS, S3...
    * @param force If true, overwrite already existing file
    * @return URI representation of the target location
    * @throws water.api.FSIOException when writing fails
    */
-  public URI exportBinaryModel(String location, boolean force) throws IOException {
+  public final URI exportBinaryModel(String location, boolean force, ModelExportOption... options) throws IOException {
```

Comment:
should we also convert `force` param into a `ModelExportOption`?

Reply:
I think that would make sense but it would also break the backwards compatibility. I think it is better to keep as-is instead of introducing a backwards compatible version of the method.

Reply:
we could keep the original method and make it call `exportBinaryModel(location, Option.FORCE);` that would make it backwards compatible, to me it seems like the right thing to do, but feel free to ignore

Reply:
thanks, I tried it your way but it actually made the existing invocations from the code to look worse - I had to do an union of the Force option and the other options. It also made me realize that the options are made more for what goes into the stream/file instead of _where_ it goes. It seems like they are not good for where it goes because that is controlled by the Persist layer and that one takes argument in a different way.